client/src/components/menubar/index.js

```diff
@@ -419,6 +432,21 @@ class MenuBar extends React.Component {
             />
           </Tooltip>
         </div>
+        <Tooltip
+          content="View the centroids for all values"
```

Comment:
```suggestion
          content="Label categories on the graph"
```

Reply:
"Centroids" is the abstraction that we use internally for dev. This doesn't really mean anything to external folks :)

Reply:
I think it should be categorical values to not confuse with the high-level categories, let me know if you disagree

Reply:
@sidneymbell this is what I have it changed to currently: "Label categorical values on the graph"